subsys/net/lib/sntp/Kconfig

```diff
@@ -4,7 +4,7 @@
 menuconfig SNTP
 	bool "SNTP (Simple Network Time Protocol)"
 	depends on NET_SOCKETS
-	depends on NET_SOCKETS_POSIX_NAMES || POSIX_API
+	depends on NET_SOCKETS_POSIX_NAMES || POSIX_API || ARCH_POSIX
```

Comment:
IMO it is much better to enable POSIX_API config option in application that adding ARCH_POSIX dependency

Reply:
I believe, that CONFIG_POSIX_API for native posix app makes no effect